lib/OpenQA/Schema/ResultSet/Screenshots.pm

```diff
@@ -21,26 +21,21 @@ use warnings;
 
 use base 'DBIx::Class::ResultSet';
 
-use DateTime;
 use OpenQA::Utils 'log_debug';
-use Try::Tiny;
 
 sub populate_images_to_job {
     my ($self, $imgs, $job_id) = @_;
 
     # insert the symlinks into the DB
     my %ids;
-    my $now = DateTime->now;
     for my $img (@$imgs) {
         log_debug "creating $img";
-        try {
-            $ids{$img} = $self->create({filename => $img, t_created => $now})->id;
-        }
-        catch {
-            my $err = $_;
-            die $err unless $err =~ /screenshots_filename/;
-            $ids{$img} = $self->find({filename => $img})->id;
-        };
+        my $dbh       = $self->result_source->schema->storage->dbh;
+        my $statement = $dbh->prepare(
```

Comment:
The convention would have been to call `$statement` here `$sth` instead.

Reply:
I have nothing against spelling things out. It makes it easier for newcomers who don't know the conventions to read the code (especially in a dynamically typed language where you can not easily deduce from the type what the object represents).

Reply:
`$sth` is a very specific convention though that you will find in all [related documentation](https://metacpan.org/pod/DBI#SYNOPSIS).

Reply:
Also notice how `$dbh` is used above.

Reply:
https://github.com/os-autoinst/openQA/pull/2600